webapp/src/components/admin_settings/jitsi_settings.tsx

```diff
@@ -59,22 +60,20 @@ const JitsiSettings = ({id, value, disabled, config, onChange, setSaveNeeded}: P
     const selectedMode = value?.usejaas ? JAAS_MODE : JITSI_MODE;
 
     let selectedSettings = DEFAULT_SETTINGS;
-    if (!value) {
-        selectedSettings = useMemo(() => ({
-            jitsiurl: config.PluginSettings.Plugins[pluginId].jitsiurl ?? DEFAULT_SETTINGS.jitsiurl,
-            jitsiappsecret: config.PluginSettings.Plugins[pluginId].jitsiappsecret ?? DEFAULT_SETTINGS.jitsiappsecret,
-            jitsiappid: config.PluginSettings.Plugins[pluginId].jitsiappid ?? DEFAULT_SETTINGS.jaasappid,
-            jitsicompatibilitymode: config.PluginSettings.Plugins[pluginId].jitsicompatibilitymode ?? DEFAULT_SETTINGS.jitsicompatibilitymode,
-            jitsiembedded: config.PluginSettings.Plugins[pluginId].jitsiembedded ?? DEFAULT_SETTINGS.jitsiembedded,
-            jitsijwt: config.PluginSettings.Plugins[pluginId].jitsijwt ?? DEFAULT_SETTINGS.jitsijwt,
-            jitsilinkvalidtime: config.PluginSettings.Plugins[pluginId].jitsilinkvalidtime ?? DEFAULT_SETTINGS.jitsilinkvalidtime,
-            jitsinamingscheme: config.PluginSettings.Plugins[pluginId].jitsinamingscheme ?? DEFAULT_SETTINGS.jitsinamingscheme,
-            usejaas: DEFAULT_SETTINGS.usejaas,
-            jaasappid: DEFAULT_SETTINGS.jaasappid,
-            jaasapikey: DEFAULT_SETTINGS.jaasapikey,
-            jaasprivatekey: DEFAULT_SETTINGS.jaasprivatekey
-        }), [config]);
-    }
+    selectedSettings = useMemo(() => ({
```

Comment:
Earlier the code was that the `selectedSettings` will be dependent on the `value` prop. Why did you change that logic?

Reply:
A condition is present after selectedSettings in line 78 for that....So we don't need to put that here.

Reply:
Yeah right....I dont think there is any need of that

Reply:
okay